ush/forecast_det.sh

```diff
@@ -74,30 +74,8 @@ WW3_det(){
 
 CICE_det(){
 	echo "SUB ${FUNCNAME[0]}: Run type determination for CICE"
-	export histfreq_n=$FHOUT
-
```

Comment:
ush/forecast_postdet.sh defines histfreq_n now.  Its not actually used in either the current feature/coupled-crow or in this branch so it could even be removed from ush/forecast_postdet.sh as well.   The MOM6 portion was removed because it was just a copy of the ice (it even says create ice_in which is an ice not ocean update) and either way, the variables that are defined are never used, so removing it seemed like a good idea for cleaning up to me.

Reply:
@JessicaMeixner-NOAA OK understand. Some explanation here: This section is designed this way for the purpose of setting the determination logic of each model components: For example, it could support FV3 is warm start and MOM6 is cold start. If such feature is not needed in the predictable future, then these sections could be left blank.

Reply:
@JianKuang-NOAA at some point there will be a need to distinguish at the minimum for CICE if it's a 'hot' or 'cold' start for CICE.  But these variables are also being determined in ush/forecast_postdet.sh and the variables are different names runtyp (which is never used) versus RUNTYPE which is used, but the logic for determining if CICE is a 'cold' or 'hot' start is not set yet.  Perhaps the logic in lines 658 through 670 that are currently in ush/forecast_postdet.sh should eventually be fixed and put in ush/forecast_det.sh?   Can this be delayed or does this need to be fixed now?

Reply:
It seems like right now CICE could only cold start. I think it makes more sense when you need to decide, at runtime, if this is a hot start or cold start. So let's delay it.